webcompat/static/js/lib/issue-wizard-popup.js

```diff
@@ -0,0 +1,119 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+function Popup() {
+  this.init = function() {
```

Comment:
This class seems to be responsible for both popup and slider. Is there a way to separate them? Perhaps could create a separate Slider class?

Reply:
@ksy36 Ksenia, the popup and slider were separate in my initial commit. @johngian asked me to combine them, which I did. See [screenshot](https://www.screencast.com/t/T34o99zOrCAF). Can you and @johngian discuss, and let me know what your common decision is?

Reply:
@munteanu210 please use links to previous comment instead of screenshots.  The screenshot might go away and we will loose context. Links are first class citizen for discussions. 😉 

Here's the link. https://github.com/webcompat/webcompat.com/pull/2937#discussion_r327076243

for this text.
> Also is there any reason not to combine them to one file just to avoid making things even more complicated with the 2 different codebases?

PS: to get the exact link, you just need to click on the time information located after the name of the commenter. Hope it helps.

Reply:
My impression for [@johngian comment](https://github.com/webcompat/webcompat.com/pull/2937#discussion_r327076243) is that it is about files name conventions and organizations, while @ksy36 is about logical code.

Reply:
Ah thanks for pointing this out, I have missed that comment @munteanu210.

So yeah I agree with your initial approach, when a single class has single responsibility it's easier to maintain later on. 

@johngian we could perhaps leave them in separate files? If you think it will complicate things, we could move all js files for the new form to a designated folder. Let me know what you thoughts are :)

Reply:
I don't have a strong opinion between single/multiple files. I just want to make sure that the naming is consistent enough so we can isolate the 2 codebases.